ui-components/src/locales/ar.json

```diff
@@ -1156,6 +1156,7 @@
         "call": "مكالمة",
         "cancel": "يلغي",
         "confirm": "يتأكد",
+        "contactPropertyManagement":"اتصل بإدارة الممتلكات",
```

Comment:
nit - add a space after the `:` here and in the other string files

Reply:
Updated